.github/workflows/worker-deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20"
+      - uses: actions/checkout@v4
+      - uses: cloudflare/wrangler-action@v3
+        with:
+          wranglerVersion: "3.57.0"
+          apiToken: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          secrets: |
```

Comment:
@gentlementlegen Do we need to pass `CLOUDFLARE_ACCOUNT_ID` here ([related PR](https://github.com/ubiquity/ubiquibot-kernel/pull/63))?

Reply:
Not required, but safer so yes it would be nice to add it here.

Reply:
@rndquu done